.github/workflows/cla.yml

```diff
@@ -21,12 +21,14 @@ jobs:
           path-to-signatures: 'signatures/version1/cla.json'
           path-to-document: 'https://safe.global/cla'
           # branch should not be protected
-          branch: 'cla-signatures'
+          branch: 'main'
```

Comment:
Why this change? We don't want the signatures to be stored in our main branch

Reply:
They will not be stored in the safe-eth-py repo, but in a centralised repo as we are doing with other repos that is specified in the following line ` remote-repository-name: 'cla-signatures'` (https://github.com/safe-global/cla-signatures)